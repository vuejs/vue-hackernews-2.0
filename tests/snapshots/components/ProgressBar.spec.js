import { mount } from '@vue/test-utils';
import ProgressBar from '@/components/ProgressBar.vue';

describe('snapshot - Spinner.vue', () => {
    // test that the progress bar is rendering in the same way as before
    it('renders correctly', () => {
        const wrapper = mount(ProgressBar);
        expect(wrapper.element).toMatchSnapshot();
    })
});
