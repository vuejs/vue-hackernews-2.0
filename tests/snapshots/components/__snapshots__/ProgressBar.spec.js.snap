// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot - Spinner.vue renders correctly 1`] = `
<div
  class="progress"
  style="width: 0%; height: 2px; background-color: rgb(255, 202, 43); opacity: 0;"
/>
`;
